packages/xarc-webpack/src/partials/babel.ts

```diff
@@ -6,9 +6,10 @@ const assign = require("lodash/assign");
 const babelLoader = require.resolve("babel-loader");
 
 import { loadXarcOptions } from "../util/load-xarc-options";
-
+const { detectXARCPath } = require("@xarc/app-dev/lib/utils");
 module.exports = function(options) {
   const xarcOptions = loadXarcOptions();
+  const xarcPath = detectXARCPath(xarcOptions.options.cwd);
```

Comment:
as discussed, `detectXARCPath` is just `path.resolve` and is rather extraneous here for example.